tests/models/test_teacher_forcing_logits.py

```diff
@@ -6,6 +6,39 @@
 import shap
 
 
+def test_falcon():
+    transformers = pytest.importorskip("transformers")
+    requests = pytest.importorskip("requests")
+    name = "fxmarty/really-tiny-falcon-testing"
+    try:
+        tokenizer = transformers.AutoTokenizer.from_pretrained(name)
+        model = transformers.AutoModelForCausalLM.from_pretrained(
+            name, trust_remote_code=True, load_in_8bit=True, device_map="auto"
```

Comment:
Ah, I noticed the tests fail on CI:

```
ImportError: Using `low_cpu_mem_usage=True` or a `device_map` requires Accelerate: `pip install accelerate`
```
  
I have a bias against adding new dev dependencies if at all possible. Can we load this model without "loc_cpu_mem_usage" : perhaps without the `load_in_8bit` kwarg?

Reply:
done, let's see if the tests pass